### PR TITLE
FilterEvents Interface Issues

### LINKED ISSUE #12702: FilterEvents UI
This issue was originally [TRAC 11864](http://trac.mantidproject.org/mantid/ticket/11864)

We have a FilterEventsUI that could be useful but needs some work

* No Help
* Doesn't seem to do FastLogs time at sample correction (or at least doesn't list it)
* Not easy to figure out how to work with, and I know the EventFiltering side of things

If this is done, should be done in conjunction with Wenduo. 

 - - - - 
Keywords: Student

### PR DESCRIPTION
**Pull request by shahrozahmed1**
Fixes #12702

Fixed interface window size, created help button which should guide to user to the FilterEventGUI documentation #13254 and enables fast log to utilised

### CODE REVIEW

**Comment by shahrozahmed1** [comment]
After investigating it seems that the previous commit seems to have fixed the issue,

**To Test**

- Open Interfaces/Utility/FilterEvents *(On windows it would only work in release mode)*
- Browse File `ExternalData/Testing/Data/DocTest/CNCS_7860_event.nxs` *(Ideally any big events nexus file)* & click Load
- Select `proton_charge` from `Sample Log` combo-box in `Filter By Log` section on bottom
- Scroll down in `Filter By Log` section and Click `Filter` button

It should produce a TableWorkspace called CNCS_7860_event_splitters

- now Tick `Fast Log` in `Advanced Setup` section on FilterEvents Interface
-  Scroll down in `Filter By Log` section and Click `Filter` button

This should convert the `CNCS_7860_event_splitters` TableWorkspace to MatrixWorkspace as [FastLog](http://docs.mantidproject.org/nightly/algorithms/GenerateEventsFilter-v1.html#properties) should

**Comment by DanNixon** [comment]
A couple of things that may as well be fixed alongside this:
- The window title is the default "MainWindow"
- When the position of the sliders on the right are changed there are log messages displayed at notice level, this could be because there are `print`statements in there or it is actually using the logger at `notice` level, either way this should at least be set to `debug`

File: Code/Mantid/scripts/FilterEvents/eventFilterGUI.py [hunk 1]
```diff
@@ -518,7 +529,8 @@ def move_lowerSlider(self):
     def set_minLogValue(self):
         """ Set the starting time and left slide bar
         """
-        print "Minimum Log Value = %s" %(str(self.ui.lineEdit_5.text()))
+        error_msg = "Minimum Log Value = %s" %(str(self.ui.lineEdit_5.text()))
+        Logger("Filter_Events").error(error_msg)
```
Comment (DanNixon) [inline Code/Mantid/scripts/FilterEvents/eventFilterGUI.py]: Why is this at error level?

File: Code/Mantid/scripts/FilterEvents/eventFilterGUI.py [hunk 2]
```diff
@@ -357,7 +363,8 @@ def set_startTime(self):
 
         # Convert to integer slide value
         ileftvalue = int( (newtime0-xlim[0])/(xlim[1] - xlim[0])*100 )
-        print "iLeftSlide = %d" % (ileftvalue)
+        info_msg = "iLeftSlide = %s" % str(ileftvalue)
+        Logger("Filter_Events").information(info_msg)
```
Comment (DanNixon) [inline Code/Mantid/scripts/FilterEvents/eventFilterGUI.py]: Should this be debug?

**Pull request merged**